Week4/BlockbusterRecycler/app/src/main/java/com/e/blockbusterrecycler/MovieRepo.kt

```diff
@@ -0,0 +1,109 @@
+package com.e.blockbusterrecycler
+
+
+object MovieRepo {
```

Comment:
Nice job on this class overall, and you've done well to include the text into the strings file. In an app that has more than just hardcoded data this could be problematic but for this application it works well, and it was creative to use a string instead of a list of stars, it prevents having to include a list of stars in the model, although if this app were to expand it might be good to see things like a list of stars as a list, and have a model class for the stars as well, as celebrities star in multiple movies. For this hard coded interface this does work.